doc/input-output-reference/src/overview/group-thermal-zone-description-geometry.tex

```diff
@@ -3819,11 +3781,11 @@ \subsubsection{Surface Window Inside Face Glazing Net Infrared Heat Transfer Rat
 
 \subsubsection{Surface Window Shortwave from Zone Back Out Window Heat Transfer Rate {[}W{]}}\label{surface-window-shortwave-from-zone-back-out-window-heat-transfer-rate-w}
 
-This is the short-wave radiation heat transfer from the zone back out the window in watts. This is a measure of the diffuse short-wave light (from reflected solar and electric lighting) that leave the zone through the window. This output variable is the term called ``{[}Short-wave radiation from zone transmitted back out the window{]}'' under the description above for Surface Window Heat Gain Rate output variable.
+This is the short-wave radiation heat transfer from the zone back out the inside face of the window in watts. This is a measure of the diffuse short-wave light (from beam and diffuse solar and electric lighting) that leaves the zone through the window.
```

Comment:
Update Surface Window Shortwave from Zone Back Out Window Heat Transfer Rate to add interior incident beam (from FullInterior solar distribution) and explain the inside face boundary volume.